package.json

```diff
@@ -44,29 +44,30 @@
         "browserWatch": "npm run browserWatch -w packages/toolkit"
     },
     "devDependencies": {
-        "eslint": "^8.56.0",
-        "eslint-config-prettier": "9.1",
-        "eslint-plugin-aws-toolkits": "file:plugins/eslint-plugin-aws-toolkits",
-        "eslint-plugin-header": "^3.1.1",
-        "eslint-plugin-unicorn": "^50.0.1",
-        "@typescript-eslint/eslint-plugin": "^5.59.0",
-        "@typescript-eslint/parser": "^5.59.1",
         "@types/vscode": "^1.68.0",
         "@types/vscode-webview": "^1.57.1",
+        "@types/webpack-env": "^1.18.1",
+        "@typescript-eslint/eslint-plugin": "^5.59.0",
+        "@typescript-eslint/parser": "^5.59.1",
         "@vscode/codicons": "^0.0.33",
         "@vscode/test-electron": "^2.3.8",
         "@vscode/test-web": "^0.0.50",
         "@vscode/vsce": "^2.19.0",
-        "@types/webpack-env": "^1.18.1",
+        "eslint": "^8.56.0",
+        "eslint-config-prettier": "9.1",
+        "eslint-plugin-aws-toolkits": "file:plugins/eslint-plugin-aws-toolkits",
+        "eslint-plugin-header": "^3.1.1",
+        "eslint-plugin-unicorn": "^50.0.1",
+        "husky": "^9.0.7",
         "prettier": "^2.8.8",
         "prettier-plugin-sh": "^0.12.8",
         "pretty-quick": "^3.1.3",
-        "husky": "^9.0.7",
-        "typescript": "^5.0.4",
         "ts-node": "^10.9.1",
+        "typescript": "^5.0.4",
         "webpack": "^5.83.0",
         "webpack-cli": "^5.1.4",
-        "webpack-dev-server": "^4.15.1"
+        "webpack-dev-server": "^4.15.1",
+        "webpack-merge": "^5.10.0"
```

Comment:
We just added this new package, the other changes are just things being automatically reordered.